hi-ml-cpath/src/health_cpath/utils/naming.py

```diff
@@ -68,6 +68,7 @@ class MetricsKey(str, Enum):
     ACC_MACRO = 'macro_accuracy'
     ACC_WEIGHTED = 'weighted_accuracy'
     CONF_MATRIX = 'confusion_matrix'
```

Comment:
you could maybe deprecate the previous plots and replace it by your plots with numbers and percentages

Reply:
Can I do this in another PR?

Reply:
sure, I'm just a bit sceptical about adding an option without corresponding code?  those plots options are tied to DeepMILPlotsHandler, you should update save_plots function with this new option

Reply:
Oh my bad I thought you added this to plots options, nevermind